pyCardDeck/standard_deck.yml

```diff
@@ -0,0 +1,61 @@
+!!python/object:pyCardDeck.deck.Deck
+_cards:
+- !!python/object:pyCardDeck.cards.PokerCard {name: Seven of Spades, rank: '7', suit: Spades}
```

Comment:
Does this know whih object to construct Card from? What happens if a different Class for the card is provided?